### DISCUSSION TITLE
PSD support?

### DISCUSSION THREAD

**Discussion by hokutonanto** [sylikc/jpegview#228] [General]
Is PSD format planned?
JpegView is near perfect for me, PSD reading it's the only thing missing.

**Comment by qbnu** [comment]
I'll try to implement it at some point, or at least add the ability to view the embedded thumbnail.

**Comment by hokutonanto** [comment]
That's great! Thanks, @qbnu !

**Comment by sylikc** [comment]
added in https://github.com/sylikc/jpegview/pull/233 and merged

available in 1.3.46
